chef/cookbooks/crowbar/templates/default/crowbar-post-upgrade.sh.erb

```diff
@@ -66,11 +66,21 @@ if [[ -f /sbin/drbdadm ]] ; then
         log "Waiting for DRBD to bring the devices up..."
         sleep 1
     done
+    drbdadm status
+
+    # Wait for replication to start
+    while ! drbdadm status | grep -qi replication; do
+        log "Replication did not start yet: `drbdadm status`"
+        sleep 1
+    done
+    drbdadm status
+
     # Now wait after DRBD data are synced with the other node
-    while drbdadm status | grep -q Inconsistent; do
+    while drbdadm status | grep -qi -e inconsistent -e replication; do
```

Comment:
OK, I've added the output from the real upgrade to the PR description. That also shows us that simple grepping for "uptodate" will not help us, as that string is already present during replication.

@nick-wang do you have an idea for a simple check after the replication was done, to make sure it ended correctly? Does it make sense to check the exit value of some command? Or could we expect some special error output?